app/views/components/input_yes_no.scala.html

```diff
@@ -36,8 +36,7 @@
     <fieldset class="inline" id="@{field.id}">
 
         <legend id="@{field.id}-legend">
-            @if(legendAsHeading){<h1 class="legend-as-heading">}
-            <span id="@{field.id}-id" @if(legendClass.nonEmpty){class="@legendClass"}>@label</span>
+            @if(legendAsHeading){<h1 id="@{field.id}-id" class="legend-as-heading heading-large">@label}
```

Comment:
Looking at the recommended markup on the story I don't think we need the @if(legendAsHeading) check at all now?